codejail/tests/test_jail_code.py

```diff
@@ -52,12 +54,8 @@ def test_with_log_messages(self, log_log):
     return text
 
 
-class JailCodeHelpers(object):
+class JailCodeMixin(helpers.JailMixin):
```

Comment:
I'm good with subclassing UnitTest for the mixin class, since it defines a `setUp()` method, but here's some evidence that subclassing `object` is a common practice.